### ISSUE TITLE
Argus dataset: Create validation set split

### ISSUE THREAD

**Issue by pasky** [brmson/dataset-sts#22]
We need that for proper reporting, split from the training set.

**Comment by pasky** [comment]
Done some time ago.
